docs/develop/customize.md

```diff
@@ -22,3 +24,43 @@ Passing the `--no-install-js` option, skips re-installation of JS dependencies.
 Go to the browser [*https://localhost:5000/*](https://localhost:5000) or refresh the page. And voilà! The logo has been changed!
 
 **WARNING**: If you do not see it changing, check in an incognito window, the browser might have cached the logo.
+
+## Change colors
+
+You might also been wondering: *How do I change the colors so I can make my instance look like my institution's theme?*. We show what changes you need to do in order to achieve that.
+
+We are going to change the top header section in the frontpage to apply our custom background color. Open the `assets/scss/<your instance shortname>/variables.scss` file and edit it as below:
+
+``` console
+$navbar_background_image: unset;
+$navbar_background_color: #000000; // Note this hex value is an example. Choose yours.
+```
+
+Then, run the `invenio-cli update --no-install-js` command as above and refresh the page! You should be able to see your top header's color changed! You can find all the available variables that you can change [here](https://github.com/inveniosoftware/invenio-app-rdm/blob/master/invenio_app_rdm/theme/assets/scss/invenio_app_rdm/variables.scss).
+
+## Change search result
+
+Changing how your results are presented in the search page is also something quite common. We show how you can do that.
```

Comment:
I think we will have to tone down on the "we show how to do it" :sweat_smile: . The pattern is obvious by now:

```suggestion
Changing how your results are presented in the search page is also something quite common.
```

Reply:
Yeah, I agree. From friendly to bossy is a thin line heh